presentation/src/main/java/com/pocs/presentation/model/post/BasePostEditUiState.kt

```diff
@@ -10,6 +10,7 @@ interface BasePostEditUiState {
     val category: PostCategory
     val currentUserType: UserType
     val onlyMember: Boolean
+    val isUserAnonymous : Boolean
```

Comment:
이렇게하면 중복으로 이것을 구현하는 두 객체에서 구현해야하기 때문에 이 인터페이스에 아래와 같이 넣는게 좋습니다.
```suggestion
    val isUserAnonymous get() = currentUserType == UserType.ANONYMOUS
```